ChiModules/LinearBoltzmannSolver/lbs_03f_resetswporderings.cc

```diff
@@ -50,11 +51,12 @@ void LinearBoltzmann::Solver::ResetSweepOrderings(LBSGroupset& groupset)
   MPI_Allreduce(&local_app_memory,&max_proc_memory,
                 1,MPI_DOUBLE,MPI_MAX,MPI_COMM_WORLD);
 
-  chi_log.Log(LOG_0)
-    << "\n" << std::setprecision(3)
-    << "           Total application memory (max): "
-    << total_app_memory/1000.0 << " GB\n"
-    << "           Maximum process memory        : "
-    << max_proc_memory/1000.0 << " GB\n\n";
+  if (options.verbose_inner_iterations)
+    chi_log.Log(LOG_0)
+      << "\n" << std::setprecision(3)
+      << "           Total application memory (max): "
+      << total_app_memory/1000.0 << " GB\n"
```

Comment:
Tiny nitpick, but this should be 1024 (and below for the max).

Reply:
Made the change.